docs/quickstart-docker.md

```diff
@@ -5,93 +5,99 @@ Percona XtraBackup works in combination with a MySQL-compatible database, Percon
-    |`--password`   | Prompts the user to enter the password for the root user. For convenience you can add the password for the root user to this option, for example, `--password=secret`. Then the password will be passed automaticaly when running the command. Note that if you specify the password with `--password=secret`, the password is visible in `docker ps`, `docker ps -a` (docker history) and regular ps command.          |
-    | `8.0.34` | Use this tag to specify a specific version. Avoid using the `latest` tag. <br> In our example, we use the `8.0.34` tag. In Docker, a tag is a label assigned to an image and is used to maintain different versions of an image.|
+If you do not add a tag, Docker uses `latest` as the default tag and downloads the newest image from [percona/percona-xtrabackup on the Docker Hub](https://hub.docker.com/r/percona/percona-xtrabackup). This image can be in a different series or version from what you expect since the latest image changes over time. The usage of tags ensures that you use [compatible versions](quickstart-overview.md#limitations) of Percona Server for MySQL and Percona XtraBackup.
 
-    If you do not add a tag, Docker uses `latest` as the default tag and downloads the newest image from [percona/percona-xtrabackup on the Docker Hub](https://hub.docker.com/r/percona/percona-xtrabackup). This image can be in a different series or version from what you expect since the latest image changes over time. The usage of tags ensures that you use [compatible versions](quickstart-overview.md#limitations) of Percona Server for MySQL and Percona XtraBackup.
+Starting with Percona XtraBackup 8.0.35-31, you can run the Docker ARM64 version of Percona XtraBackup. Use the `8.0.35-31-aarch64` tag instead of `8.0.35-31`.
 
-    To connect the Percona XtraBackup container to the Percona Server container, we recommend using the `–-user root` option in the Docker command.
+### Connect the Percona XtraBackup container to the Percona Server container 
+
+We recommend using the `–-user root` option in the Docker command.
     
-    * Run a Docker container example
+Run a Docker container example
 
-        ```{.bash data-prompt="$"}
-        $ docker run --name pxb --volumes-from psmysql -v backupvol:/backup_8034 -it --user root percona/percona-xtrabackup:8.0.34 /bin/bash -c "xtrabackup --backup --datadir=/var/lib/mysql/ --target-dir=/backup_8034 --user=root --password; xtrabackup --prepare --target-dir=/backup_8034"
-        ```
+```{.bash data-prompt="$"}
+$ docker run --name pxb --volumes-from psmysql -v backupvol:/backup_8034 -it --user root percona/percona-xtrabackup:8.0.34 /bin/bash -c "xtrabackup --backup --datadir=/var/lib/mysql/ --target-dir=/backup_8034 --user=root --password; xtrabackup --prepare --target-dir=/backup_8034"
+```
 
-        You are prompted to enter the password, which is `secret`, in our example.
+You are prompted to enter the password, which is `secret`, in our example.
```

Comment:
Sentence doesn't make sense.
You are prompted to enter a password. In our example, this password is `secret`.

Reply:
Fixed